docs/infrastructure_and_maintenance/cache/http_cache/content_aware_cache.md

```diff
@@ -291,15 +291,16 @@ In the event when a new version of `Child` is published, the following keys are
 In summary, HTTP Cache for any location representing `[Child]`, any Content that relates to the Content `[Child]`, the 
 location for `[Child]`, any children of `[Child]`, any Location that relates to the Location `[Child]`, location for
 `[Parent1]`, any children on `[Parent1]`.
-Effectively, in this example HTTP cache for `[Parent1]` and `[Child]` will be cleared.
+Effectively, in this example HTTP cache for `[Parent1]` and `[Child]` are cleared.
 
 
 ### Tags purged on move event
 
 With the same Content structure as above, the `[Child]` location is moved below `[Parent2]`.
 
-The new structure will then be:
-```
+The new structure is then:
+
+```bash
```

Comment:
```suggestion
```
```

In this code block we don't have any particular language so I would leave it as plain text

Reply:
Vale reacts with a warning for posting code blocks without a highlighter. I am leaving it as you suggested.